vuepress/docs/v7.1/docs/create/blueprint-features.md

```diff
@@ -3,69 +3,54 @@ sidebarDepth: 2
 ---
 # Entando Blueprint Features
 
-The Entando Blueprint is an easier and faster way to customize your application by generating controllers, repositories, services, and micro frontends for your entity. The project files are generated in minutes, by entering data that define the parameters of your application. 
+This page examines the features of the Entando Blueprint, which is an easier and faster way to customize an Entando Application by generating controllers, repositories, services, and micro frontends for an entity. The project files are autogenerated once the application parameters are specified. 
```

Comment:
"customize an Entando Application" doesn't feel right since you don't modify the Entando App. With the ACP language, perhaps "create components for an Entando Application including a Spring Boot microservice and corresponding micro frontends". We can drop controllers and repositories since those are internal to SpringBoot.